src/util/getShape1D.ts

```diff
@@ -19,7 +19,9 @@ export function getShape1D(kind: ShapeKind, shapeOptions = {}) {
     case 'pseudoVoigt':
       return new PseudoVoigt(shapeOptions);
     default:
+      // eslint-disable-next-line no-case-declarations
+      const unHandled: never = kind;
       // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
-      throw new Error(`getShape1D: unknown shape kind: ${kind}`);
+      throw Error(`Unknown distribution ${unHandled}`);
```

Comment:
why did you change these? there are no typescript errors

Reply:
The idea is to make exhaustive this switch, so if we add more options in ShapeKind and do not add the respective cases in the switch, the compiler throws us an error. I am moving the error from the runtime to the compiletime

Reply:
okay, so in that case, wrap the code in a block instead of disabling the eslint error: `default: { ... }`

Reply:
you can also try to enable the `switch-exhaustiveness-check` rule from typescript-eslint